processed 4 tasks

init:
A: object(0,0)

task 1 'publish'. lines 9-21:
Error: Transaction Effects Status: Iota Move Bytecode Verification Error. Please run the Iota Move Verifier for more information.
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: IotaMoveVerificationError, source: Some("_::m. 'init' cannot be 'entry'"), command: Some(0) } }

task 2 'set-address'. lines 23-23:
Error: INVALID TEST. Unknown object, object(1,0)

task 3 'run'. lines 25-40:
Error: Error checking transaction input objects: DependentPackageNotFound { package_id: 0x0000000000000000000000000000000000000000000000000000000000000000 }
